src/three/math/OBB.js

```diff
@@ -43,6 +44,32 @@ export class OBB {
 
 	}
 
+	// returns boolean indicating whether the ray has intersected the obb
+	intersectsRay( ray ) {
+
+		_ray.copy( ray ).applyMatrix4( this.inverseTransform );
+		return _ray.intersectsBox( this.box );
```

Comment:
If you wish to follow `three` interface for `OBB#intersectsRay`, it should be of type `Ray -> boolean`. You could also remove code duplication.

```suggestion
		return _ray.intersectRay( this.box, _norm ) !== null;
```

Reply:
> it should be of type Ray -> boolean

What do you mean by this? `Ray.intersectsBox` already [returns a boolean value](https://threejs.org/docs/?q=ray#api/en/math/Ray.intersectsBox)

Reply:
My bad, I read `intersectBox`.